docs/docs/configuration.md

```diff
@@ -39,6 +39,14 @@ If none of these files exist, sku will use its built-in default configuration.
 
 Config files can use either TypeScript or JavaScript.
 
+## bundler
+
+Type: `string`
+
+Default: `webpack`
+
+The bundler that sku uses to build the application.
+
```

Comment:
I feel the type for this one should specify that this value only accepts a union `webpack` or `vite`

May also be worth adding a link to the Vite docs in here

```suggestion

Type: `webpack` | `vite`

Default: `webpack`

The bundler that sku uses to build the application.
See [Vite support](https://seek-oss.github.io/sku/#/./docs/vite) for details.
```

Or something along those lines